packages/flutter_template_images/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 4.0.1
```

Comment:
I think technically this should be 4.1.0? Doesn't really matter, but I'm more used to thinking in semver now :)

Reply:
Yeah I wondered about that too since we're adding something that if we take away would be breaking... but to be honest am still a little fuzzy on how well semver maps to assets. Updated.

Reply:
The division isn't super crisp, but the official description is:

>MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

This is new functionality to the package, rather than fixing a bug in the package.